sys/arduino/include/serialport.hpp

```diff
@@ -84,7 +84,7 @@ class SerialPort {
      *
      * @param[in] speed     speed in bits per second (baud)
      */
-    void begin(int speed);
+    void begin(unsigned long speed);
```

Comment:
Wait, didn't @haukepetersen say this should be long? Arduino reference [is confirming this](https://www.arduino.cc/en/Serial/Begin)